<html>

<IMG width="200" height="50" SRC="fig/helse-sor-ost.jpg" ALT="[HSØ LOGO]" valign="top" >
<IMG width="260" height="50" SRC="fig/uio_segl_a.png" ALT="[UIO LOGO]" valign="top">
<IMG width="260" height="50" SRC="fig/oslo-universitetssykehus-ous-logo.png" ALT="[OUS LOGO]" valign="top">
<head>
   <meta HTTP-EQUIV="Content-Type" CONTENT="text/html; 
charset=iso-8859-1">
   <meta NAME="GENERATOR" CONTENT="Mozilla/4.06 [en] (X11; I; HP-UX 
B.10.20 9000/778) [Netscape]">
   <title>BPB3: BayesPI-BAR version 3 package User Guide </title>
</head>
<body TEXT="#000000" BGCOLOR="#FFFFFF" LINK="#FF0000" VLINK="#000099" 
ALINK="#FE0000" BACKGROUND="<&quot;">

<h1 style="background-color:DodgerBlue;"> 
<font COLOR="#FFFFFF" > BPB3: BayesPI-BAR version 3 package User Guide </font></h1>
<!-- black #333300
white #FFFFFF
-->
<P>
<br>
<A HREF="mailto:junbai.wang@medisin.uio.no">Junbai Wang</a>* </b>, MingYi Yang**, Magnar Bjoras .....  <br><br> 
<b>
*Department of Clinical Molecular Biology (EpiGen), Univesity of Oslo <br>
**Department of Microbiology, Oslo University Hospital 
<br>
</b>

<P>
<B>Abstract</B>
<P>
BayesPI-BAR3 (or bpb3) is Bayesian method for protein–DNA interaction with binding affinity Ranking in Python3. 
It is a command line tool, which not only includes all R functions of previous BayesPI-BAR [1] and Python2 modules of BayesPI-BAR2[2,3], 
but also add a number of new functions and modules in the new bpb3 package (e.g., preprocess of download data from ICGC data portal, 
generate heatmap for predicted significant TF binding affinity changes, estimate significant TF binding affinity changes by using 
either clustered PWMs [5] or a two levels approach, and clean temporary files et al.) Especially, the previous BayesPI-BAR programs 
were implemented in Python2 and scatted to various command line scripts, which may not be suited for users with limited knowledge in Python. 
Bpb3 is upgraded to Python3 and can be easily installed in either Linux or MAC OS system, which can be simply called by users as 
the other build-in commands in the operating system. In particular, bpb3 provides a precompiled input parameter configure file, 
in which user can modify various parameters associated with running bpb3 pipeline such as input/output file location and disable/enable certain steps in the pipeline. 
Then, the parameter configure file can be directly loaded into bpb3 package for running the user desired calculation automatically. 
It significantly simplifies the application of bpb3 in different tasks. Demos of these new functions and examples are provided in the package.

</P>
<P>
The work is supported by
<A href="http://www.helse-sorost.no" target = "new" > 
South-Eastern Norway Regional Health Authority </a>
<br>
<br>
<B>References </B>
<br>
1.<A href="http://nar.oxfordjournals.org/content/early/2015/07/21/nar.gkv733.abstract" target ="new">
BayesPI-BAR: a new biophysical model for characterization of regulatory sequence variations</A>
<br> Junbai Wang and Kirill Batmanov<br> Nucleic Acids Research 2015, July 
<br><br>

2. <A href="http://rdcu.be/uEib" target ="new" >
Integrative whole-genome sequence analysis reveals roles of regulatory mutations in BCL6 and BCL2 in follicular lymphoma.
</A><br>
Batmanov K, Wang W, Bjoras M, Delabie J, Wang J.
<br>
Sci Rep. 2017 Aug 1;7(1):7040. doi: 10.1038/s41598-017-07226-4.
<br><br>
 
3. <A href="https://www.frontiersin.org/articles/10.3389/fgene.2019.00282/abstract" target = "new"> 
BayesPI-BAR2: a new Python package for predicting functional non-coding mutations in cancer patient cohorts. </A> 
<br> K. Batmanov , J. Delabie and JB Wang. <br> Front Genet. 2019 Apr 2;10:282.
<br><br>

4. <A href="https://doi.org/10.1016/j.csbj.2020.09.038" target ="new"> 
HMST-Seq-Analyzer: A new python tool for differential methylation and hydroxymethylation analysis in various DNA methylation sequencing data. </A>
<br>A. Farooq, S. Grønmyr, O. Ali, T. Rognes, K. Scheffier, M. Bjørås, JB. Wang. <br>
Computational and Structural Biotechnology Journal. 2020 September.
<br><br>

5.<A href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-022-04615-z" target = "new">
abc4pwm: affinity based clustering for position weight matrices in applications of DNA sequence analysis.</A>
<br> Omer Ali, Amna Farooq, Mingyi Yang, Victor X. Jin, Magnar Bjørås and Junbai Wang.<br>
BMC Bioinformatics. 2022 March. <br><br>

6. <A href="https://doi.org/10.1016/j.csbj.2022.03.023" target= "new">
Integrating whole genome sequencing, methylation, gene expression, topological associated domain information in regulatory mutation prediction: a study of follicular lymphoma.  </A>
<br>
Amna Farooq, Gunhild Trøen, Jan Delabie, Junbai Wang.<br>
Computational and Structural Biotechnology Journal. 2022 March.
<br><br> 

</body> </html>




